app/components/planning_applications/search_component.html.erb

```diff
@@ -34,13 +34,25 @@
         <div class="govuk-accordion__section-header">
           <h3 class="govuk-accordion__section-heading">
             <button type="button" class="govuk-accordion__section-button">
-              <%= t(".status_filter", selected: selected_status_count, total: total_status_count) %>
+              Filters
```

Comment:
How about improving and showing how many application types filters are selected as well?

Reply:
I think this was no longer needed so i've removed it but can confirm with the POs